pkg/controller/controller.go

```diff
@@ -805,11 +842,13 @@ func (c *Controller) syncHandler(key string) error {
 			dashdeployment, err = c.kubeclientset.AppsV1().Deployments(apimanager.Namespace).Update(pattern1.WorkerDeployment(apimanager, y,worknum))
 		}
 
-		//for instance mysql deployment
-		if apimanager.Spec.Replicas != nil && *apimanager.Spec.Replicas != *mysqldeployment.Spec.Replicas {
-			//y:= pattern1.AssignMysqlConfigMapValues(apimanager,configmap)
-			klog.V(4).Infof("Apimanager %s replicas: %d, deployment2 replicas: %d", name, *apimanager.Spec.Replicas, *mysqldeployment.Spec.Replicas)
-			mysqldeployment, err = c.kubeclientset.AppsV1().Deployments(apimanager.Namespace).Update(mysql.MysqlDeployment(apimanager))
+		if useMysqlPod {
+			//for instance mysql deployment
+			if apimanager.Spec.Replicas != nil && *apimanager.Spec.Replicas != *mysqldeployment.Spec.Replicas {
+				//y:= pattern1.AssignMysqlConfigMapValues(apimanager,configmap)
```

Comment:
Shall we remove the commented code y:=...

Reply:
Fixed with https://github.com/wso2/k8s-wso2am-operator/pull/77